src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
+@keyframes rotates {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
 body {
+  width: 100%;
+  height: 100%;
   margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  position: absolute;
+  inset: 0;
+  margin: auto;
+
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minute-hand {
+    margin: auto;
+    position: absolute;
+    left: inherit;
+    right: 0;
+    top: 20vmin;
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    z-index: -1;
+
+    animation-name: rotates;
+    transform-origin: 1.5vmin 20vmin;
+    animation-duration: 3600s;
+    animation-timing-function: steps(60);
+    animation-iteration-count: infinite;
```

Comment:
The use of `animation-iteration-count: infinite;` is causing the animation to run indefinitely. If this is the intended behavior, it's fine. However, if you want the animation to run a certain number of times, you can specify the number instead of `infinite`.